### PR TITLE
add deploymendMode as a recipe option

### PR DESCRIPTION
**Pull request by t00f**
Adds a `deploymentMode` in the recipe options:
- `Unrestricted` = no constraint
- `Unique` = Unique in the current namespace and its sub namespaces
- `NamespaceUnique` = Unique in the current namespace only

> Fixes https://github.com/aporeto-inc/aporeto/issues/1860
> Project: https://github.com/orgs/aporeto-inc/projects/623

### CODE REVIEW

File: specs/+recipeoptions.spec [hunk 1]
```diff
@@ -20,3 +20,14 @@ attributes:
     - JSON
     - YAML
     default_value: JSON
+
+  - name: deploymentMode
+    description: Indicates if the recipe can be run multiple time.
```
Comment (primalmotion) [inline specs/+recipeoptions.spec]: can you explain what each value means?

Comment (primalmotion) [reply specs/+recipeoptions.spec]: actually this should be in the recipe itself,Options are for api users, not the backend

Comment (t00f) [reply specs/+recipeoptions.spec]: Yes, I kinda hesitated which is why I made the PRs live :D Will do !

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "7c15a11fa89aabc0ee28e717dacf99410ed6b988"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "39077dc42813b83632034fb3cda1d6d7cee55f3d"
  },
  {
    "project": "fix_1860",
    "component": "clad",
    "pr-id": "1294",
    "commit-sha": "36f175413c964aa418e796f1fd1595f1989cf621"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "clad",
    "pr-id": "1294",
    "commit-sha": "650f6ca11be794801df20a66414fe0f194e039fe"
  },
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "b17439148d09e8d131bc96a7385f9ef95eff53d3"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "53769d4a04218c6c8d2fd553187337b03574ff8b"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "clad",
    "pr-id": "1294",
    "commit-sha": "6118b12301b085c1d94ee6109bdd20f567127f0b"
  },
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "340e187fbd55ef51e395121240176f8dc6911de8"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "45b101a709a920943132eb41d9f2a9298c364176"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "1767db3f8e1e4b82ed63293333eab5cb3bc85b9a",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "clad",
    "pr-id": "1294",
    "commit-sha": "ef0257f1950934605d94084c10631ec1aae4776c",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "f51f3eb6a4354e7fc736e7961724efd221e3ad27",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "cf85a8f3716d420f2dcbb4383f2d0a37fe05a8b2",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "f51f3eb6a4354e7fc736e7961724efd221e3ad27",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "clad",
    "pr-id": "1294",
    "commit-sha": "ef0257f1950934605d94084c10631ec1aae4776c",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "cf85a8f3716d420f2dcbb4383f2d0a37fe05a8b2",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "401a0419ac4146074f22c235473b108b8464ed37",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "cf85a8f3716d420f2dcbb4383f2d0a37fe05a8b2",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "401a0419ac4146074f22c235473b108b8464ed37",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "f51f3eb6a4354e7fc736e7961724efd221e3ad27",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "f51f3eb6a4354e7fc736e7961724efd221e3ad27",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "cf85a8f3716d420f2dcbb4383f2d0a37fe05a8b2",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "bada9d31deeb4ec588df300cb4c7f67348a12e79",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "cf85a8f3716d420f2dcbb4383f2d0a37fe05a8b2",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "9894ce54784708d4f5901965f7cb1593b99c4317",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "f51f3eb6a4354e7fc736e7961724efd221e3ad27",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "c7da7e68b91e043eb1223a863d5b15cea002113a",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "f51f3eb6a4354e7fc736e7961724efd221e3ad27",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "cf85a8f3716d420f2dcbb4383f2d0a37fe05a8b2",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "cf85a8f3716d420f2dcbb4383f2d0a37fe05a8b2",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "3725c6861cc76025ec95fa49a7daacc3aa4035a5",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "f51f3eb6a4354e7fc736e7961724efd221e3ad27",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "1b32c03543d1085abedbc0bcfcf208b037a2ad20",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "64b4dc5276909b19fc69c88331607ada782a442d",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "09867e3e06a0a8fafc0ab07d4322ac67f7198e1b",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "d162b4f41b9d472c78af26e57e13a8a75474d3ea",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "64b4dc5276909b19fc69c88331607ada782a442d",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "09867e3e06a0a8fafc0ab07d4322ac67f7198e1b",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "09867e3e06a0a8fafc0ab07d4322ac67f7198e1b",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "91d5e322c3d7b973813f0785a3fc940abf571b6d",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "64b4dc5276909b19fc69c88331607ada782a442d",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "09867e3e06a0a8fafc0ab07d4322ac67f7198e1b",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "9ee995787d937ac4ab36f586c9cf7cb239cf935a",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "64b4dc5276909b19fc69c88331607ada782a442d",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "09867e3e06a0a8fafc0ab07d4322ac67f7198e1b",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "9ee995787d937ac4ab36f586c9cf7cb239cf935a",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "5df75745f189d216c2b1da75218772bf29a221d2",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "9ee995787d937ac4ab36f586c9cf7cb239cf935a",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "5df75745f189d216c2b1da75218772bf29a221d2",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "09867e3e06a0a8fafc0ab07d4322ac67f7198e1b",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "a57b2055ba4d60deed22ef8588eed08754a7eb86",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "c240b63d80c0293268d3bea8eb1435252e2ca127",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "b5c9011437038237cb5c5c5988c12708da1c6770",
    "pipeline": "master"
  }
]
```

File: specs/recipe.spec [hunk 2]
```diff
@@ -33,6 +33,21 @@ model:
 # Attributes
 attributes:
   v1:
+  - name: deploymentMode
+    description: |-
+      Indicates if the deployment mode of the recipe. If Unrestricted, recipe
+      can be deployed multiple times. If Unique, only one deployment is allowed
+      in the current namespace and its child namespaces. If NamespaceUnique, only
```
Comment (primalmotion) [inline specs/recipe.spec]: > only one deployment is allowed in the current namespace and its child namespaces

what about the parents?

File: specs/importreference.spec [hunk 3]
```diff
@@ -26,6 +26,10 @@ model:
   - '@named'
   - '@timeable'
 
+# Indexes
+indexes:
+- - label
```
Comment (primalmotion) [inline specs/importreference.spec]: seems you need

- - namespace
   - label

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "899a6d7fee79b011b2f2f90c2238d5547c0a456d",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "c240b63d80c0293268d3bea8eb1435252e2ca127",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "b5c9011437038237cb5c5c5988c12708da1c6770",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "6bd05ab16e24ea26bcf64af21d6438cfca9f429a",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "6fda2eeeedda16c61891cce56c454ceafd0effed",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "40dddd679acdbb5c034eeed1ee20a59e5aefa66b",
    "pipeline": "master"
  }
]
```

File: specs/importreference.spec [hunk 4]
```diff
@@ -76,3 +81,25 @@ attributes:
         - - $identity=processingunit
     extensions:
       refMode: pointer
+
+  - name: importConstraint
```
Comment (primalmotion) [inline specs/importreference.spec]: can we rename this `constraint` ?

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "d5eddd38ebab8c4c603c1b548c2b2320f309dbfc",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "3f10ac7097caa82c599d3939bf11727276959902",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "40dddd679acdbb5c034eeed1ee20a59e5aefa66b",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "c801f3057a4c881f7d68af267bf35b4dfe735228",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "5b7c344220b8353f6c14970d29c7be03b93ffbf1",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "b28d1f8ab708b95a7795b0298d7dd1536bba5370",
    "pipeline": "master"
  }
]
```

**Comment by npylai** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "voila",
    "pr-id": "97",
    "commit-sha": "c5ec9ae12f5323a4e8fd4ad375ebec46874d267c",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "c801f3057a4c881f7d68af267bf35b4dfe735228",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "5b7c344220b8353f6c14970d29c7be03b93ffbf1",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "b28d1f8ab708b95a7795b0298d7dd1536bba5370",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "apotests",
    "pr-id": "2523",
    "commit-sha": "06778c64fe08aa0799ea5bd0f2e11d4b9088f491",
    "pipeline": "master"
  }
]
```

**Comment by npylai** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "c801f3057a4c881f7d68af267bf35b4dfe735228",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "5b7c344220b8353f6c14970d29c7be03b93ffbf1",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "b28d1f8ab708b95a7795b0298d7dd1536bba5370",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "apotests",
    "pr-id": "2523",
    "commit-sha": "4b021fb7242a200edfec3e13cdf0e0e6aca0921e",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "voila",
    "pr-id": "97",
    "commit-sha": "c5ec9ae12f5323a4e8fd4ad375ebec46874d267c",
    "pipeline": "master"
  }
]
```

**Comment by npylai** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "c801f3057a4c881f7d68af267bf35b4dfe735228",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "5b7c344220b8353f6c14970d29c7be03b93ffbf1",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "b28d1f8ab708b95a7795b0298d7dd1536bba5370",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "apotests",
    "pr-id": "2523",
    "commit-sha": "4b021fb7242a200edfec3e13cdf0e0e6aca0921e",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "voila",
    "pr-id": "97",
    "commit-sha": "c5ec9ae12f5323a4e8fd4ad375ebec46874d267c",
    "pipeline": "master"
  }
]
```

**Comment by npylai** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "c801f3057a4c881f7d68af267bf35b4dfe735228",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "5b7c344220b8353f6c14970d29c7be03b93ffbf1",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "b28d1f8ab708b95a7795b0298d7dd1536bba5370",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "apotests",
    "pr-id": "2523",
    "commit-sha": "4b021fb7242a200edfec3e13cdf0e0e6aca0921e",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "voila",
    "pr-id": "97",
    "commit-sha": "c5ec9ae12f5323a4e8fd4ad375ebec46874d267c",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "apotests",
    "pr-id": "2523",
    "commit-sha": "4b021fb7242a200edfec3e13cdf0e0e6aca0921e",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "voila",
    "pr-id": "97",
    "commit-sha": "c5ec9ae12f5323a4e8fd4ad375ebec46874d267c",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "c801f3057a4c881f7d68af267bf35b4dfe735228",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "5b7c344220b8353f6c14970d29c7be03b93ffbf1",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "b28d1f8ab708b95a7795b0298d7dd1536bba5370",
    "pipeline": "master"
  }
]
```

**Comment by npylai** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "c801f3057a4c881f7d68af267bf35b4dfe735228",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "5b7c344220b8353f6c14970d29c7be03b93ffbf1",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "b28d1f8ab708b95a7795b0298d7dd1536bba5370",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "apotests",
    "pr-id": "2523",
    "commit-sha": "3f45ac58e77668bc666a7edf2121f144ac3b1e29",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "voila",
    "pr-id": "97",
    "commit-sha": "c5ec9ae12f5323a4e8fd4ad375ebec46874d267c",
    "pipeline": "master"
  }
]
```

**Comment by npylai** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "6aca9be2953bc051dd58cd5dcc7f3432612e57d0",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "5b7c344220b8353f6c14970d29c7be03b93ffbf1",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "b28d1f8ab708b95a7795b0298d7dd1536bba5370",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "voila",
    "pr-id": "97",
    "commit-sha": "c5ec9ae12f5323a4e8fd4ad375ebec46874d267c",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "apotests",
    "pr-id": "2523",
    "commit-sha": "3f45ac58e77668bc666a7edf2121f144ac3b1e29",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "6aca9be2953bc051dd58cd5dcc7f3432612e57d0",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "5b7c344220b8353f6c14970d29c7be03b93ffbf1",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "b28d1f8ab708b95a7795b0298d7dd1536bba5370",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "voila",
    "pr-id": "97",
    "commit-sha": "c5ec9ae12f5323a4e8fd4ad375ebec46874d267c",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "apotests",
    "pr-id": "2523",
    "commit-sha": "9df5709042cfa718635478ba05809e289a81b5a1",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "890619b1fddd37f545abdfc0e5487a5a3cf2225d",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "voila",
    "pr-id": "97",
    "commit-sha": "9fbde18e4ced392640a85130df2cb8d1c7753f39",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "apotests",
    "pr-id": "2523",
    "commit-sha": "4c7c02e1a3e582a2fccdc5d023dba5db126db122",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "6aca9be2953bc051dd58cd5dcc7f3432612e57d0",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "1015de9aff3359aa20644b79e383414f6194c19a",
    "pipeline": "master"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "6aca9be2953bc051dd58cd5dcc7f3432612e57d0",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "1015de9aff3359aa20644b79e383414f6194c19a",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "890619b1fddd37f545abdfc0e5487a5a3cf2225d",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "apotests",
    "pr-id": "2523",
    "commit-sha": "4c7c02e1a3e582a2fccdc5d023dba5db126db122",
    "pipeline": "master"
  }
]
```

**Comment by npylai** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "apotests",
    "pr-id": "2523",
    "commit-sha": "4c7c02e1a3e582a2fccdc5d023dba5db126db122",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "6aca9be2953bc051dd58cd5dcc7f3432612e57d0",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "1015de9aff3359aa20644b79e383414f6194c19a",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "890619b1fddd37f545abdfc0e5487a5a3cf2225d",
    "pipeline": "master"
  }
]
```

**Comment by npylai** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "6aca9be2953bc051dd58cd5dcc7f3432612e57d0",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "1015de9aff3359aa20644b79e383414f6194c19a",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "890619b1fddd37f545abdfc0e5487a5a3cf2225d",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "apotests",
    "pr-id": "2523",
    "commit-sha": "8a526fb607cd261abcd45aa547ac76d846d62c80",
    "pipeline": "master"
  }
]
```

**Comment by npylai** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "6aca9be2953bc051dd58cd5dcc7f3432612e57d0",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "1015de9aff3359aa20644b79e383414f6194c19a",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "890619b1fddd37f545abdfc0e5487a5a3cf2225d",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "apotests",
    "pr-id": "2523",
    "commit-sha": "3327d9123172737036e03a8d06613f289e625a84",
    "pipeline": "master"
  }
]
```

**Comment by npylai** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "6aca9be2953bc051dd58cd5dcc7f3432612e57d0",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "1015de9aff3359aa20644b79e383414f6194c19a",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "890619b1fddd37f545abdfc0e5487a5a3cf2225d",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "apotests",
    "pr-id": "2523",
    "commit-sha": "3327d9123172737036e03a8d06613f289e625a84",
    "pipeline": "master"
  }
]
```

**Comment by npylai** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "6aca9be2953bc051dd58cd5dcc7f3432612e57d0",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "1015de9aff3359aa20644b79e383414f6194c19a",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "890619b1fddd37f545abdfc0e5487a5a3cf2225d",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "apotests",
    "pr-id": "2523",
    "commit-sha": "da28a606a420e8fa5de6d750eb4f1d2d00e49ec7",
    "pipeline": "master"
  }
]
```

**Comment by npylai** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix_1860",
    "component": "gaia",
    "pr-id": "404",
    "commit-sha": "6aca9be2953bc051dd58cd5dcc7f3432612e57d0",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "backend",
    "pr-id": "544",
    "commit-sha": "1015de9aff3359aa20644b79e383414f6194c19a",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "underwater",
    "pr-id": "82",
    "commit-sha": "890619b1fddd37f545abdfc0e5487a5a3cf2225d",
    "pipeline": "master"
  },
  {
    "project": "fix_1860",
    "component": "apotests",
    "pr-id": "2523",
    "commit-sha": "da28a606a420e8fa5de6d750eb4f1d2d00e49ec7",
    "pipeline": "master"
  }
]
```

**Pull request merged**